### DISCUSSION TITLE
Reasons for the `remote` keyword

### DISCUSSION THREAD

**Discussion by dabrahams** [hylo-lang/hylo#1514] [Language design]
We've forgotten why `remote let` and `remote inout` can't just be spelled `let` and `inout` in the positions where they are currently allowed. We should retrieve that knowledge or try dropping the keyword.  This discussion can record the rationale persistently.

**Comment by kyouko-taiga** [comment]
Now I remember!
```hylo
fun f(x: let Int) { print(x) }
```
Is `x` an `Int` or a remote `Int`?

Another thing to consider while we're on the topic is that we need to syntaxes, one for value terms and one for type terms. Otherwise we'll run into the same ambiguities as the one we'd have if tuples terms and types were both spelled with parentheses.

Currently we have that `[let x]` is a remote type expression of type `remote let Int`, given `x: Int`.

**Reply by kyouko-taiga** [reply]
Also related: https://github.com/hylo-lang/hylo/issues/1326
